pkgs/applications/misc/octoprint/default.nix

```diff
@@ -82,6 +82,12 @@ let
 
               # requires octoprint itself during tests
               doCheck = false;
+              patches =  [
+              (pkgs.substituteAll {
+                src = ./vgencmd.patch;
+                vcgencmd = "${self.pkgs.libraspberrypi}/bin/vcgencmd";
+              })
+            ];
```

Comment:
Can't we just do
```
  substituteInPlace octoprint_pi_support/__init__.py --replace /usr/bin/vcgencmd ${self.pkgs.libraspberrypi}/bin/vcgencmd
```
in postPatch

Reply:
It should theoretically work like that, too. Shall I change it?

Reply:
That would be cleaner to read and easier to maintain.

Reply:
ok